packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -1169,4 +1169,28 @@ void main() {
     final EditableText editableText = tester.widget(find.byType(EditableText));
     expect(editableText.magnifierConfiguration, equals(myTextMagnifierConfiguration));
   });
+
+  testWidgets('Error color for cursor while validation', (WidgetTester tester) async {
```

Comment:
Nit: "while validation" => "during validation" or "while validating"

Reply:
Updated the desc